invenio_app_rdm/theme/templates/invenio_app_rdm/search/header.html

```diff
@@ -12,38 +12,36 @@
 {%- block navbar %}
 <div class="ui fluid container">
   <div class="ui fluid container search-header">
-    <div
-      class="ui container relaxed middle aligned grid"
-      style="padding: 2em 0px;"
-    >
+    <div class="ui container relaxed middle aligned grid" style="padding: 2em 0px;">
       <div class="row">
         {% block search_left %}
-            <div class="three wide column">
-            {%- block brand %}
-              {%- if config.THEME_LOGO %}
-                <a href="/"><img class="ui image" src="{{ url_for('static', filename=config.THEME_LOGO)}}" alt="{{_(config.THEME_SITENAME)}}" /></a>
-              {%- else %}
-                <a class="navbar-brand" href="/">{{_(config.THEME_SITENAME)}}</a>
-              {%- endif %}
-            {%- endblock brand%}
-            </div>
+        <div class="three wide column">
+          {%- block brand %}
+          {%- if config.THEME_LOGO %}
+          <a href="/"><img class="ui image" src="{{ url_for('static', filename=config.THEME_LOGO)}}"
+              alt="{{_(config.THEME_SITENAME)}}" /></a>
+          {%- else %}
+          <a class="navbar-brand" href="/">{{_(config.THEME_SITENAME)}}</a>
+          {%- endif %}
+          {%- endblock brand%}
+        </div>
         {% endblock search_left%}
         {%- block search_middle %}
-            <div class="nine wide column">
-            <div class="ui container">
-                {{ render_searchbar('search-bar') }}
-            </div>
-            </div>
+        <div class="nine wide column">
+          <div class="ui container">
+            {{ render_searchbar('search-bar') }}
+          </div>
+        </div>
         {% endblock search_middle %}
         {% block search_right %}
-            <div class="four wide column">
-              <div class="ui container">
-                  <div class="ui buttons">
-                  <a class="ui button" role="button" href="/deposits">Uploads</a
-                  ><a class="ui button" role="button" href="/coming-soon">Communities</a>
-                  </div>
-              </div>
+        <div class="four wide column">
+          <div class="ui container">
+            <div class="ui buttons">
+              <a class="ui button" role="button" href="/deposits">Uploads</a><a class="ui button" role="button"
+                href="/coming-soon">Communities</a>
             </div>
+          </div>
+        </div>
         {% endblock %}
       </div>
     </div>
```

Comment:
[potential shelf]: Most of this header can now be the same as the header for the rest of the site. The only difference should be the searchbar itself which would be a react app... although ideally it would be a react app everywhere and only on the search page would it have a slightly different behaviour.

Reply:
Yes, I agree with this. I just wanted to wait till deposit is cleaned from the folder. Then clean search. Then see how we can only have one header, potentially some things should go to invenio-assets/theme. We are needed too much overwriting atm. IMHO.

Reply:
> We are needed too much overwriting atm. IMHO

yes.

I hope we don't get ourselves too muddied up with all of our temporary fixes...